src/library/scala/collection/SeqView.scala

```diff
@@ -112,6 +114,36 @@ object SeqView {
     @throws[IndexOutOfBoundsException]
     def apply(i: Int) = underlying.apply(i)
   }
+
+  @SerialVersionUID(3L)
+  class Sorted[A, B >: A](underlying: SomeSeqOps[A])(implicit ord: Ordering[B]) extends SeqView[A] {
+    private[this] lazy val _sorted = {
+      val len = underlying.length
+      new SeqView.Id(
```

Comment:
Do we need the `Sorted.Id` wrapper here? Is it providing some extra laziness over and above just 

```scala
lazy val _sorted: Array[A] = ...
```
?

Reply:
uh, no, it's not. I forgot to change that

Reply:
using an `Array` doesn't quite work as, at least with the current code, it doesn't always produce an `Array`. I suppose that's changeable, but I'm not sure if it really matters

Reply:
Yeah I'm sure `Iterable[A]` is fine